terraform/development/main.tf

```diff
@@ -33,12 +33,39 @@ locals {
 module "database" {
   source = "../modules/rds"
 
-  prefix                            = local.prefix
-  allocated_storage                 = 5
-  db_subnet_group_name              = module.networking.db_private_subnet_group_name
-  ingress_source_security_group_ids = []
-  instance_class                    = "db.t3.micro"
-  vpc_id                            = module.networking.vpc_id
+  prefix                             = local.prefix
+  allocated_storage                  = 5
+  db_subnet_group_name               = module.networking.db_private_subnet_group_name
+  ingress_from_ecs_security_group_id = module.ecs.ecs_security_group_id
+  instance_class                     = "db.t3.micro"
+  vpc_id                             = module.networking.vpc_id
+}
+
+module "ecs" {
+  source = "../modules/ecs"
+
+  prefix = local.prefix
+  additional_task_role_policy_arns = {
+    "RDS_access" : module.database.rds_data_access_policy_arn
+    "Redis_access" : "arn:aws:iam::aws:policy/AmazonElastiCacheFullAccess"
+  }
+  ecr_repository_url                = "815624722760.dkr.ecr.eu-west-2.amazonaws.com/core-ecr"

```

Comment:
I don't think I can output this from meta, so had to hardcode for now. Not found a better way yet (other than tucking it away in `tfvars` or similar)